src/main/java/com/parsing/model/ResultReportItem.java

```diff
@@ -0,0 +1,49 @@
+package com.parsing.model;
+
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.Id;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.Table;
+import jakarta.persistence.ManyToOne;
+import jakarta.persistence.JoinColumn;
+import jakarta.persistence.FetchType;
+import lombok.AllArgsConstructor;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+import lombok.Getter;
+import lombok.ToString;
+
+import java.math.BigDecimal;
+import java.util.UUID;
+
+@Entity
+@Setter
+@Getter
```

Comment:
You are using @Data and @Getter & @Setter. Let's use only one.

Reply:
Can't see @Data